src/PresentationalComponents/ImpactBadge/ImpactBadge.js

```diff
@@ -0,0 +1,85 @@
+import { QuestionIcon, SecurityIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React from 'react';
+
+class ImpactBadge extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.colorList = {
+            default: 'var(--pf-global--Color--200)', // grey
```

Comment:
Can you move this to styles file? I don't really like using inline styles, but you can use something like `inc-c-critical-badge`

Reply:
I'm just using color property from https://github.com/patternfly/patternfly-react/tree/master/packages/react-icons. Should I use the external CSS definition anyway?

Reply:
Css definitions would be slightly better, but I guess you can keep it this way for simplicity.